### ISSUE TITLE
[BUG] when recovery fails or is hard-stopped don't create and empty wallet.

### ISSUE THREAD

**Issue by justfordaniel** [casinocoin/tokenwallet-mobile#128]
**Describe the bug**
on recovery with seeds it does not recover and creates a new wallet if you put a space after the email address

**To Reproduce**
recover wallet and put a space after the email address

**Expected behavior**
it should ignore the space and recover the wallet

**Screenshots**
n/a

**Smartphone (please complete the following information):**
i assume this is irrelevant but anyhow
 - Device: sony xa1 (G3121)
 - OS: android
 - OS Version 8.0.0
 - App Version wallet 0.1.10

**Additional context**

**Comment by t3nshi** [comment]
You are missing a lot on the process to make it validate. I tried to follow your single step and was not able to reproduce your issue. Please give clear steps on the exact things you did or tried so I can replicate on my phone. According to your description this does not seem to be a device specific issue. So I should be able to see the same on my phone. I created a wallet  (on testnet) then I used the words to restore such wallet adding many spaces at the end of the email and it was recovered. It may be that a crucial step was skipped on the process in the way you just mentioned it. Please check the exact steps one would need to take so I can reproduce your issue and provide a full solution. Thanks for your testing and the time you are taking to report these.

**Comment by justfordaniel** [comment]
i only added one space after the email but it was not on test net....its a real wallet....maybe that will make a difference?

**Comment by justfordaniel** [comment]
right.....my mistake

it is doing something else

what i had done before was close the wallet because i thought it was not going to find it..... it was taking a very long time.

so if you close it while its processing and then open and login.....

your wallet is recovered as the seed is there but there is no balances ect so it looks like a new wallet

**Comment by t3nshi** [comment]
It is not very clear what you are describing. It is best that you don't give this abstract and ambiguous descriptions, so you provide instead a concise and very concrete set of steps saying in what order you do what and what is the unexpected result against the expected restult. Let me see if I am getting this right because your persistence on not following the template provided is not helping me understand your problem:

**Describe the bug**
In recovery with seeds, when I interrupted the process abruptly by closing the wallet, when I opened it again is not showing any account even though it shows the correct seed.


**To Reproduce**
Steps to reproduce the behavior:
1. Go to Recovery from mnemonic words page
2. Fill all form and start recovery process
3. During recovery process force close the app
4. Open the app again, and see it is asking for pin now
5. log in and see there are not any account recovered and no balance.



**Expected behavior**
??? not sure what you expected... 

Please If forgive me if I am not understanding your problem correctly. If this is aproximate just correct it or modify it a bit so I understand. If it is way different, now you get why this templates are provided, then entlighten me providing more accurate and concrete description. 

Thanks in advance for your effort and consideration.

**Comment by justfordaniel** [comment]
yes that is correct.  i did those steps because i thought it was not working.

however, if i go to view seed words.  they are all there as they should be

**Comment by t3nshi** [comment]
Well in this case can you tell me what you expected to happen? since you force quit the app on recovery it is not something we can really avoid, and this is not really a use case. In my opinion it is a flop, and from the login screen the user can restore the wallet again and maybe let the process complete this time. 
You can provide any ideas anyway, Let me know what you think should be expected from the app in this case so we discuss it.

**Comment by justfordaniel** [comment]
you're correct, it can just be repeated so maybe its not an issue.  it was just confusing for me

ideas

if it flops like this maybe it should not create an account that you can log in to after?  instead it could have create/recover options again.... instead of login/recover options

or 

it has the seeds if you click to view them so i guess when it reopens it could continue 

or

the processing window could say that this could take up to 2mins or something so impatient people wait longer lol

**Comment by t3nshi** [comment]
I think those ideas are good and worth exploring. Let me talk about it internally. I let you know what we think about it soon

**Comment by justfordaniel** [comment]
k thanks

**Comment by t3nshi** [comment]
We decided that we will try to make the app realize when a wallet has no accounts and provide an error message and present us with alternative to recover a wallet or create a new one. For a case like the one you presented it will activate when you try to open the wallet. Let's wait for the implementation of this and we test and try to reproduce this to see if it fits.

**Comment by justfordaniel** [comment]
sounds good
